subprojects/architecture-test/src/changes/archunit_store/internal-api-nullability.txt

```diff
@@ -3037,7 +3037,10 @@ Class <org.gradle.internal.os.OperatingSystem$Solaris> is not annotated (directl
 Class <org.gradle.internal.os.OperatingSystem$Unix> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (OperatingSystem.java:0)
 Class <org.gradle.internal.os.OperatingSystem$Windows> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (OperatingSystem.java:0)
 Class <org.gradle.internal.os.OperatingSystem> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (OperatingSystem.java:0)
+Class <org.gradle.internal.problems.DefaultProblemDiagnosticsFactory$DefaultProblemDiagnostics> is not annotated (directly or via its package) with @org.gradle.api.NonNullApi in (DefaultProblemDiagnosticsFactory.java:0)
```

Comment:
🤔 Can you add a `package-info.java` with `@NonNullApi` to the new package?

Reply:
Will do. I thought I had but I guess I missed that package.

Reply:
Done